src/scenes/home/termsOfService/termsOfService.js

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from 'react';
+import Section from 'shared/components/section/section';
+
+class Terms extends Component {
+  render() {
+    return (
+      <div>
```

Comment:
Because you are using a Section component, you do not need to wrap within a div, the Section component will act as the wrapper here.